debian/changelog

```diff
@@ -1,3 +1,34 @@
+daos (2.7.101-17) unstable; urgency=medium
+  [ Cedric Koch-Hofer]
+  * Add support of the libasan to the mercury dependencies
+
+ -- Cedric Koch-Hofer <cedric.koch-hofer@hpe.com>  Mon, 27 Oct 2025 14:12:00 +0100
+
+daos (2.7.101-16) unstable; urgency=medium
+  [ Jeff Olivier]
+  * Make daos-spdk conflict with spdk
+
+ -- Jeff Olivier <jeffolivier@google.com>  Mon, 19 May 2025 14:12:00 -0700
```

Comment:
The whole debian folder is no longer used since we produce .deb files using fpm.   We shoudl just remove the folder rather than updating this file